docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -244,10 +244,26 @@ SELECT IPv6CIDRToRange(toIPv6('2001:0db8:0000:85a3:0000:0000:ac1f:8001'), 32);
 └────────────────────────────────────────────────────────────────────────┘
 ```
 
-## toIPv4(string)
+## toIPv4
 
 An alias to `IPv4StringToNum()` that takes a string form of IPv4 address and returns value of [IPv4](../data-types/ipv4.md) type, which is binary equal to value returned by `IPv4StringToNum()`.
 
+**Syntax**
+
+```sql
+toIPv4(string)
+```
+
+**Arguments**
+
+- `string` — IP address. [String](../data-types/string.md).
+
+**Returned value**
+
+- `string` converted to the current IPv4 address. [String](../data-types/string.md).
+
+**Examples**
```

Comment:
I find both examples odd. Can we simplify so that
- the first example demonstrates what `toIPv4` returns using a simple input string (as opposed to its return type) - no need to include `IPv4StringToNum` here,
- the second example may compare both functions with respect to their return type and hex-ed values.